client/src/pages/track/Track.svelte

```diff
@@ -117,92 +121,115 @@
-                    <br />
+                </Container>
+                <Container ty={ContainerType.Divider}>
+                    <h3>File Attachment</h3>
+                    <a download type={mime} href="/api/document/download?doc={trackingNumber}">
+                        <div class="column download">
+                                <DocumentDownload size={IconSize.Large} alt="Download the attached document."/>
+                                Download File Attachment 
+                        </div>
+                    </a>
+                </Container>
+                <Container ty={ContainerType.Divider}>
+                    <h3>Paper Trail</h3>
                     <table>
-                        <td><p class="header-color"><b>Paper Trail</b></p></td>
-                        <tr>
-                            <td><b>Office</b></td>
-                            <td><b>Creation</b></td>
-                            <td><b>Time Elapsed</b></td>
-                            <td><b>Action</b></td>
-                            <td><b>Remarks</b></td>
-                            <td><b>Evaluator</b></td>
-                        </tr>
-                        {#each trail as { target, creation, status, remark, name, picture, email }}
+                        <thead>
                             <tr>
-                                <td>
-                                    {#if target === null}
-                                        End
-                                    {:else}
-                                        {$allOffices[target]}
-                                    {/if}
-                                </td>
-                                <td>{creation.toLocaleString()}</td>
-                                <td>{computeTimeDiff(creation)}</td>
-                                <td>{status}</td>
-                                <td>{remark}</td>
-                                <td class="evaluator">
-                                    <img src={picture} alt={name} />
-                                    <a href="mailto:{email}">{name}</a>
-                                </td>
+                                <td>Office</td>
+                                <td>Creation</td>
+                                <td>Time Elapsed</td>
+                                <td>Action</td>
+                                <td>Remarks</td>
+                                <td>Evaluator</td>
                             </tr>
-                        {/each}
+                        </thead>
+                        <tbody>
+                            {#each trail as { target, creation, status, remark, name, picture, email }}
+                                <tr>
+                                    <td>
+                                        {#if target === null}
+                                            End
+                                        {:else}
+                                            {$allOffices[target]}
```

Comment:
This may possibly be `undefined`.

Reply:
What is possibly undefined here?

Reply:
Well, in a very rare (but dumb) case, it may happen that `target` is actually out of bounds.